docs/iidm/model/generator.md

```diff
@@ -7,17 +7,17 @@ The `com.powsybl.iidm.network.Generator` interface is used to model a generator.
 
 # Characteristics
 
-| Attribute | Type | Required | Default value | Description |
-| --------- | ---- | -------- | ------------- | ----------- |
-| EnergySource | `EnergySource` | yes | `OTHER` | The energy source |
-| MinP | double | yes | - | Minimal active power |
-| MaxP | double | yes | - | Maximum active power |
-| RegulatingTerminal | `TerminalExt` | no | - | The terminal used for regulation |
-| VoltageRegulatorOn | boolean | yes | - | The voltage regulator status |
-| TargetP | double | yes | - | The active power target |
-| TargetQ | double | no | - | The reactive power target |
-| TargetV | double | no | - | The voltage target |
-| RatedS | double | yes | - | The rated nominal power |
+| Attribute | Type | Unit | Required | Default value | Description |
+| --------- | ---- | ---- | -------- | ------------- | ----------- |
+| EnergySource | `EnergySource` | - | yes | `OTHER` | The energy source |
+| MinP | double | MW | yes | - | Minimal active power |
+| MaxP | double | MW | yes | - | Maximum active power |
+| RegulatingTerminal | `TerminalExt` | - | no | - | The terminal used for regulation |
+| VoltageRegulatorOn | boolean | - | yes | - | The voltage regulator status |
+| TargetP | double | MW | yes | - | The active power target |
+| TargetQ | double | MVAr | no | - | The reactive power target |
+| TargetV | double | kV | no | - | The voltage target |
+| RatedS | double | MVa | yes | - | The rated nominal power |
```

Comment:
Standard is MVA  (with A uppercase) I think

Reply:
you're right !